vtex-apps/organizations-challenge
# Chore/update tooling and code style

#### What does this PR do? \*

This PR adds Slots options alongside Extension Points. It also removes the use of ramda and makes the PermissionChallenge code more clear by moving each query to its own separate hook.

The PR also updates the tooling, updates CSS Handles to use the new major, and fixes TS errors on the entire app.

**I recommend reviewing commit by commit to make sense of the changes**

#### How to test it? \*

Look for this component: 
<img width="224" alt="Screen Shot 2021-02-18 at 15 43 51" src="https://user-images.githubusercontent.com/8127610/108405661-1d2d3300-7200-11eb-8a2c-ab26e4864154.png">

The changes shouldn't make any difference to this app's behavior. I'll give you the credentials to login and see the prices on slack :)

[Workspace with the changes](https://icarovtex--b2bstoreqa.myvtex.com/)
[Master workspace (before the changes)](b2bstoreqa.myvtex.com/)
